server/bin/pbench-index.py

```diff
@@ -152,16 +153,16 @@ def main(options, name):
 
     res = 0
     try:
-        ARCHIVE_rp = os.path.realpath(idxctx.config.ARCHIVE)
-        if not os.path.isdir(ARCHIVE_rp):
+        ARCHIVE_rp = Path(idxctx.config.ARCHIVE).resolve()
+        if not ARCHIVE_rp.is_dir():
             idxctx.logger.error("{}: Bad ARCHIVE={}", name, idxctx.config.ARCHIVE)
             res = 3
-        INCOMING_rp = os.path.realpath(idxctx.config.INCOMING)
-        if not os.path.isdir(INCOMING_rp):
+        INCOMING_rp = Path(idxctx.config.INCOMING).resolve()
+        if not INCOMING_rp.is_dir():
             idxctx.logger.error("{}: Bad INCOMING={}", name, idxctx.config.INCOMING)
             res = 3
-        qdir = idxctx.config.get("pbench-server", "pbench-quarantine-dir")
-        if not os.path.isdir(qdir):
+        qdir = Path(idxctx.config.get("pbench-server", "pbench-quarantine-dir"))
+        if not qdir.is_dir():
```

Comment:
resolve it?

Reply:
Perhaps `if not qdir.resolve(strict=True).is_dir()`, and capture the error raised by `.resolve()` when the path fails?